language_negotiation_lambda.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
```

Comment:
This is from the lambda blueprint code.  As a lambda there is nothing else to effect.  It is standalone.